tools/lint/src/eslint.ts

```diff
@@ -7,7 +7,7 @@ import { LintFn } from './lint.types';
 
 const rootDir = process.cwd();
 const eslintConfigPath = path.resolve(__dirname, '../config/eslint.config.js');
-export const esLintExtensions = ['js', 'ts', 'tsx'];
+export const esLintExtensions = ['ts', 'tsx'];
```

Comment:
Are we sure we want to omit and lingering `.js` files?

Reply:
i think so - only js files we should have are transpiled?